packages/wallet-management/src/wallets.ts

```diff
@@ -228,6 +229,12 @@ const exodus: Wallet = new InjectedConnector(
   (window as any).exodus?.ethereum,
 );
 
+const safe: Wallet = new SafeWalletConnector({
+  name: 'Safe',
+  installed: () => false,
```

Comment:
same here: can we add some realy logic here? to detect if its running in the safe app iframe? cause we really do use that function. and if this just returns false, we won't show the wallet

Reply:
I've made the return type of func `Promise<boolean> | boolean`